server/protocols/webadmin/webadmin-mailbox/src/main/java/org/apache/james/webadmin/routes/UserMailboxesRoutes.java

```diff
@@ -281,6 +281,7 @@ public void defineMailboxExists() {
                         .statusCode(HttpStatus.NOT_FOUND_404)
                         .type(ErrorType.INVALID_ARGUMENT)
                         .message("Invalid get on user mailboxes")
+                        .cause(new IllegalArgumentException("Mailboxes do not exist"))
```

Comment:
We are testing that one mailbox exists, not many. Should be:

`Mailbox does not exist`